### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by samuelscheit** [Flam3rboy/discord-bot-client#202] [General]
-

**Comment by Nitrrine** [comment]
Description: We help you when you have any problems.

**Reply by AdamMors11** [reply]
Hello,I dont have a pc as i dont play games really and wanted to ask if it was possible to do it on mobile to be exact,Android A7.
Thanks!

**Reply by splatterxl** [reply]
> Hello,I dont have a pc as i dont play games really and wanted to ask if it was possible to do it on mobile to be exact,Android A7.
> Thanks!

No.

**Reply by Nitrrine** [reply]
**[Support Reply]:** In my opinion, no

**Reply by gabi75018** [reply]
hello everyone !

**Comment by ghost** [comment]
How do I log out to use a different token?

**Reply by Nitrrine** [reply]
> How do I log out to use a different token?

Go to account settings and logout
When you go to default login page press CTRL + S

#195

**Reply by Nitrrine** [reply]
[Support Reply]
Working?

**Comment by MCPE2019Github** [comment]
can i use this with human's token
i mean the tokens human accs have?

**Reply by Nitrrine** [reply]
[Support Reply]
idk but you can

**Reply by MCPE2019Github** [reply]
i tried but it said to verify your phone on all the ones i tried

**Reply by Nitrrine** [reply]
[Support Reply]
Hm ok

**Reply by splatterxl** [reply]
Using human tokens is against Discord ToS (self-botting).
